go/logger/redirect_stderr_linux.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build linux,!android
+
+package logger
+
+import (
+	"os"
+	"syscall"
+)
+
+func tryRedirectStderrTo(f *os.File) error {
+	return syscall.Dup3(int(f.Fd()), 2, 0x00000008)
```

Comment:
What's this `0x00000008` flag? Should this be zero?

Reply:
The flag means File removed.  It could entirely be that it should be zero.  I wasn't sure and am not familiar enough with the codebase to know.

The flags come from the fcntl.h (or fcntl-linux.h) file on a Linux machine (usually located somewhere in /usr/include).

Reply:
Do you know the name of the flag? I'm curious where you got it, and what other places might be using it?

Reply:
bits/fcntl-linux.h:# define DN_DELETE	0x00000008	/* File removed.  */

To be clear, I wasn't entirely sure what the tryRedirectStderrTo was used, so I figured File removed fit best.  I'm definitely open to changing it to the correct flag that it should be, I just don't know what it is.  I changed it to "0" locally, and am able to use everything just fine as well.  Is there a specific test that tests this?  Running run_test.sh against the unit tests, there are a lot of failures because there's no docker so no local service is running to test against.

Reply:
I think it should be zero.